config/initializers/app_key.rb

```diff
@@ -0,0 +1,2 @@
+ENV['AWS_ACCESS_KEY_ID'] = "AKIAIKTE4V3CV6KAQRTQ"
+ENV['AWS_SECRET_ACCESS_KEY'] = "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"
```

Comment:
You have to remove this file. We can't include an `app_key.rb`. Make sure it's removed from history with `git rm --cached config/initializers/app_key.rb`